scripts/lay-down-os

```diff
@@ -26,11 +25,6 @@ device_defined()
     fi
 }
 
-format_device()
```

Comment:
With the AWS EBS volume we format, but do not partition. I'd like to keep it here. A good update in the future is to allow file system format selection.